cortex/cassandra/entrypoint-wrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ $1 = 'cassandra' ]]; then
+  # Create default keyspace for single node cluster
+  CQL="CREATE KEYSPACE cortex WITH replication = {'class':'SimpleStrategy', 'replication_factor' : 1};"
+  until echo $CQL | cqlsh; do
+    echo "cqlsh: Cassandra is unavailable - retry later"
+    sleep 2
+  done &
+fi
+
+exec /docker-entrypoint.sh "$@"
```

Comment:
@gutorc92, correct me if I'm wrong, but how could Cassandra be up and running if the Dockerfile CMD will get executed **after** the Entrypoint get's executed? Woudn't this `do..until` be in a loop forever? Thanks!

Reply:
It will not loop forever because the special caracter "&" makes the loop runs in the background so the rest of script is evaluated. The line 12 will start up  Cassandra.

Reply:
Oh ok. Thanks for pointing that out!